ios/PresentationLayer/Recipes/Sources/Recipes/Images/ImagesViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Created by David Kadlček on 12.05.2023
+//  Copyright © 2023 Matee. All rights reserved.
+//
+
+import Foundation
+import UIToolkit
+
+final class ImagesViewModel: BaseViewModel, ViewModel, ObservableObject {
+    // MARK: Dependencies
+    private weak var flowController: FlowController?
+
+    init(flowController: FlowController?) {
+        self.flowController = flowController
+        super.init()
+        setupCacheCapacity()
+    }
+    
+    private func setupCacheCapacity() {
+        URLCache.shared.memoryCapacity = 10_000_000 // ~10 MB memory space
+        URLCache.shared.diskCapacity = 1_000_000_000 // ~1GB disk cache space
+    }
```

Comment:
Maybe move to `AppDelegate` so we won't forget to call this in other apps that are based on DevStack?

Reply:
Done